README.md

```diff
@@ -81,37 +81,28 @@ lessen the impact of sudden increases and decreases of network hashing rate.
 
 **The current block reward schedule:**
 
-1–99,999: 0–1,000,000 Dogecoin
-
-100,000–144,999: 0–500,000 Dogecoin
-
-145,000–199,999: 250,000 Dogecoin
-
-200,000–299,999: 125,000 Dogecoin
-
-300,000–399,999: 62,500 Dogecoin
-
-400,000–499,999: 31,250 Dogecoin
-
-500,000–599,999: 15,625 Dogecoin
-
-600,000+: 10,000 Dogecoin
+|Block |Reward
```

Comment:
Could you make the tables non-condensed? Then it's pretty in html render and as plaintext

Reply:
Could you link me to an example of a non-condensed table?

Reply:
```
| Block          | Reward        |
| :------------- | ------------: |
| 0-100k         |     1,000,000 |
| 100k-200k      |       500,000 |
```

Reply:
Oh I see, I'll get that updated.

Reply:
@patricklodder Revision applied!